sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<p>Do a find and replace for all markdown occurrences of <code>Sagemaker</code> and change to <code>SageMaker</code>	across both notebooks in the PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Done